src/Reveal.Sdk.Dom.Tests/Visualizations/OHLCVisualizationFixture.cs

```diff
@@ -0,0 +1,275 @@
+                  }
+                },
+                "Expiration" : 1440,
+                "Bindings" : {
+                  "Bindings" : [ ]
+                }
+              },
+              "VisualizationDataSpec" : {
+                "_type" : "PivotVisualizationDataSpecType",
+                "Columns" : [ {
+                  "_type" : "DimensionColumnSpecType",
+                  "SummarizationField" : {
+                    "_type" : "SummarizationRegularFieldType",
+                    "DrillDownElements" : [ ],
+                    "ExpandedItems" : [ ],
+                    "FieldName" : "Territory"
+                  }
+                }, {
+                  "_type" : "DimensionColumnSpecType",
+                  "SummarizationField" : {
+                    "_type" : "SummarizationRegularFieldType",
+                    "DrillDownElements" : [ ],
+                    "ExpandedItems" : [ ],
+                    "FieldName" : "Conversions"
+                  }
+                }, {
+                  "_type" : "DimensionColumnSpecType",
+                  "SummarizationField" : {
+                    "_type" : "SummarizationRegularFieldType",
+                    "DrillDownElements" : [ ],
+                    "ExpandedItems" : [ ],
+                    "FieldName" : "Spend"
+                  }
+                } ],
+                "Values" : [ ],
+                "ShowGrandTotals" : false,
+                "FormatVersion" : 0,
+                "AdHocExpandedElements" : [ ],
+                "Rows" : [ ]
+              }
+            } ]
+            """;
+
+        //Act
+        var document = new RdashDocument("My Dashboard");
+
+        var excelDataSourceItem = new RestDataSourceItem("Marketing Sheet")
+        {
+            Id = "080cc17d-4a0a-4837-aa3f-ef2571ea443a",
+            Subtitle = "Excel Data Source Item",
+            Url = "http://dl.infragistics.com/reportplus/reveal/samples/Samples.xlsx",
+            IsAnonymous = true,
+            ResourceItem = new DataSourceItem
+            {
+                Id = "d593dd79-7161-4929-afc9-c26393f5b650",
+                DataSourceId = "33077d1e-19c5-44fe-b981-6765af3156a6",
+                Title = "Marketing Sheet",
+                Subtitle = "Excel Data Source Item",
+                HasTabularData = true,
+                HasAsset = false,
+                Properties = new Dictionary<string, object>
+                {
+                    { "Url", "http://dl.infragistics.com/reportplus/reveal/samples/Samples.xlsx" }
+                }
+            },
+            Fields = new List<IField>
+            {
+                new DateField("Date"),
+                new NumberField("Spend"),
+                new NumberField("Conversions"),
+                new TextField("Territory")
+            }
+        };
+        excelDataSourceItem.UseExcel("Marketing");
+
+        document.Visualizations.Add(new PivotVisualization("OHLC Visualization", excelDataSourceItem)
```

Comment:
Not sure why we add PivotVisualization here, while it's testing for OHLC Visualization